guides/common/modules/proc_adding-rhv-details-to-a-compute-profile.adoc

```diff
@@ -38,14 +38,14 @@ For each volume, enter the following details:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# hammer compute-profile create --name "{oVirt} CP"
+# hammer compute-profile create --name "oVirt CP"
 ----
 +
 . To set the values for the compute profile, enter the following command:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# hammer compute-profile values create --compute-profile "{oVirt} CP" \
+# hammer compute-profile values create --compute-profile "oVirt CP" \
```

Comment:
Unrelated, but I'm not a fan of using this appreviation. 


This would be our "default" naming scheme:
```suggestion
# hammer compute-profile values create --compute-profile "_My_oVirt_Compute_Profile_" \
```